### PR TITLE
nav bar refactor + search bar now searches categories

### PR DESCRIPTION
**Pull request by jackson-07**
search bar now searches all the items and categories & the nav bar 'if else' logic has been refactored

### CODE REVIEW

File: main_app/templates/search.html [hunk 1]
```diff
@@ -11,7 +13,7 @@ <h2>{{ searched }}</h2>
 {% for item in item %}
 <div class="card">
   <div class="card-header">
-    <a href="">Item:</a>
+    <a href="{% url 'detail' item.id %}">Item:</a>
```
Comment (MartyEternal) [inline main_app/templates/search.html:16]: Love these additions to allow for the user to click the item's link.

File: main_app/urls.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
     path('items/<int:item_id>/unassoc_category/<int:category_id>/', views.unassoc_category, name='unassoc_category'),
     
     # search bar
-    path('search_items/', views.search_items, name='search_items'),
+    path('search/', views.search, name='search'),
```
Comment (MartyEternal) [inline main_app/urls.py:28]: Updated the search_items.html to search.html. Nice

File: main_app/views.py [hunk 3]
```diff
@@ -126,16 +126,15 @@ def delete(self, request, *args, **kwargs):
       response = super().delete(request, *args, **kwargs)
       return response
 
-def search_items(request):
+def search(request):
```
Comment (MartyEternal) [inline main_app/views.py:129]: Updated to make sure search function is called properly.

File: main_app/templates/base.html [hunk 4]
```diff
@@ -1,108 +1,95 @@
...
-                <button
-                  class="btn btn-outline-success my-2 my-sm-0"
-                  type="submit"
-                >
-                  Search
-                </button>
-              </form>
+
+            <li class="nav-item">
+              <a class="nav-link" href="{% url 'history_log' %}">
```
Comment (MartyEternal) [inline main_app/templates/base.html:51]: Thanks for adding the History Log

**Review by MartyEternal — COMMENTED** [review]
Great work here. A lot of tidying up and Nav bar is now set properly with History Log and Search functions and .html file is done properly.

File: main_app/templates/base.html (hunk 4 above)
Comment (jackson-07) [reply main_app/templates/base.html:51]: You are welcome!

**Review by Exiled184 — APPROVED** [review]
nice changes

**Pull request merged**